### ISSUE TITLE
Semaphore circuit thread pool metrics regression

### ISSUE THREAD

**Issue by mattnelson** [Netflix/Hystrix#1330]
When upgrading from `1.5.3` to `1.5.5` semaphore circuits now contain a thread pool entry in the `hystrix-metrics-event-stream`. It looks like this is only a stream regression as I don't see a corresponding increase in the number of threads.

1.5.5 stream
```
data: {
  "type": "HystrixCommand",
  "name": "circuit155",
  "group": "circuit155",
  "rollingCountSemaphoreRejected": 0,
  "rollingCountThreadPoolRejected": 0,
  "propertyValue_executionIsolationStrategy": "SEMAPHORE",
  "propertyValue_executionIsolationThreadTimeoutInMilliseconds": 60000,
  "propertyValue_executionIsolationThreadInterruptOnTimeout": true,
  "propertyValue_executionIsolationThreadPoolKeyOverride": null,
  "propertyValue_executionIsolationSemaphoreMaxConcurrentRequests": 100,
  "propertyValue_fallbackIsolationSemaphoreMaxConcurrentRequests": 10,
  "threadPool": "circuit155"
}

data: {
  "type": "HystrixThreadPool",
  "name": "circuit155"
}
```

1.5.3 stream
```
data: {
  "type": "HystrixCommand",
  "name": "circuit153",
  "group": "circuit153",
  "rollingCountSemaphoreRejected": 0,
  "rollingCountThreadPoolRejected": 0,
  "propertyValue_executionIsolationStrategy": "SEMAPHORE",
  "propertyValue_executionIsolationThreadTimeoutInMilliseconds": 60000,
  "propertyValue_executionIsolationThreadInterruptOnTimeout": true,
  "propertyValue_executionIsolationThreadPoolKeyOverride": null,
  "propertyValue_executionIsolationSemaphoreMaxConcurrentRequests": 100,
  "propertyValue_fallbackIsolationSemaphoreMaxConcurrentRequests": 10,
  "threadPool": "circuit153"
}
```

**Comment by mattrjacobs** [comment]
Thanks for the report @mattnelson .  I'll get this fixed in the 1.5.6 release.  It's likely related to the refactoring I did to move metrics serialization to a single place.

**Comment by mattrjacobs** [comment]
Fixed in #1363
